pages/treeherder.py

```diff
@@ -49,6 +51,15 @@ def unchecked_repos(self):
     def unclassified_failure_count(self):
         return int(self.find_element(*self._unclassified_failure_count_locator).text)
 
+    def clear_filter(self):
```

Comment:
Actually I just realized that I no longer use this with the updated verification. For my 2 cents I wouldn't change it to add "click", but that's me.